ssh-key/Cargo.toml

```diff
@@ -60,16 +64,19 @@ std = [
     "signature/std"
 ]
 
+aes-cbc = ["dep:cbc", "encryption"]
 aes-gcm = ["dep:aes-gcm", "encryption"]
+chacha20poly1305 = ["dep:chacha20", "dep:poly1305", "encryption"]
 crypto = ["aes-gcm", "ed25519", "p256", "p384", "rsa"] # NOTE: `dsa` is obsolete/weak
 dsa = ["dep:bigint", "dep:dsa", "dep:sha1", "alloc", "signature/rand_core"]
 ecdsa = ["dep:sec1"]
 ed25519 = ["dep:ed25519-dalek", "rand_core"]
-encryption = [ "alloc", "dep:aes", "dep:bcrypt-pbkdf", "dep:ctr", "rand_core"]
+encryption = [ "alloc", "dep:aes", "dep:bcrypt-pbkdf", "dep:cbc", "dep:ctr", "rand_core"]
 getrandom = ["rand_core/getrandom"]
 p256 = ["dep:p256", "ecdsa"]
 p384 = ["dep:p384", "ecdsa"]
 rsa = ["dep:bigint", "dep:rsa", "alloc", "rand_core"]
+des = ["dep:des", "encryption"]
```

Comment:
Please put this in alphabetical order

Reply:
Also perhaps call it `3des` or `tdes` for clarity

Reply:
Renamed to tdes